app/components/article-featured-video.js

```diff
@@ -23,15 +25,27 @@ export default Component.extend(InViewportMixin,
 			'isVideoDrawerVisible:is-fixed',
 			'withinPortableInfobox:within-portable-infobox:without-portable-infobox'
 		],
-		hasTinyPlayIcon: computed.or('withinPortableInfobox', 'isVideoDrawerVisible'),
+
+		ads: inject.service(),
+		fastboot: inject.service(),
+		wikiVariables: inject.service(),
+
 		isPlayerLoading: true,
-		isPlaying: false,
+		autoplay: computed('withinPortableInfobox', function () {
+			return !this.get('fastboot.isFastBoot') &&
+				!this.get('withinPortableInfobox') &&
+				(system !== 'ios' || isSafariMinVer(10)) &&
+				$.cookie(autoplayCookieName) !== '0' &&
```

Comment:
It's set here: https://github.com/Wikia/html5-skin/blob/XW-3636/js/controller.js#L1599
We have https://wikia-inc.atlassian.net/browse/XW-3720 to make it work across communities.